pkg/deploy/controller/deploymentconfig/controller.go

```diff
@@ -135,7 +142,6 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 		// If the deployment was already created, just move on. The cache could be stale, or another
 		// process could have already handled this update.
 		if errors.IsAlreadyExists(err) {
-			c.recorder.Eventf(config, "alreadyExists", "Deployment already exists for DeploymentConfig: %s", deployutil.LabelForDeploymentConfig(config))
 			glog.V(4).Infof("Deployment already exists for DeploymentConfig %s", deployutil.LabelForDeploymentConfig(config))
```

Comment:
removed the event and retained the log statement for the case when the deployment already exists - presumably from another controller running simultaneously.